src/Cli/dotnet/commands/dotnet-help/HelpCommand.cs

```diff
@@ -23,7 +23,7 @@ public static int Run(ParseResult result)
 
             result.ShowHelpOrErrorIfAppropriate();
 
-            if (!string.IsNullOrEmpty(result.GetValue(HelpCommandParser.Argument)))
+            if (result.GetValue(HelpCommandParser.Arguments) is { } args)
```

Comment:
I don't think you need the variable name if you don't use it... or using the discard, `_`, as the name.